### PR TITLE
Fix tooltips displaying for hidden `SelectionHandler` content

### PR DESCRIPTION
**Pull request by peppy**
Split out from https://github.com/ppy/osu/pull/12626 as it got a bit more complicated due to a weird test existing. This was a pending TODO anyways, so good chance to clean things up.

The actual fix here is to move the visibility toggle away from a container with `AlwaysVisible` applied, as it's not compatible with properly hiding tooltips etc.

### CODE REVIEW

File: osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs [hunk 1]
```diff
@@ -92,21 +95,32 @@ public bool CanScaleY
...
         private OsuColour colours { get; set; }
 
         [BackgroundDependencyLoader]
-        private void load()
-        {
-            RelativeSizeAxes = Axes.Both;
-
-            recreate();
-        }
+        private void load() => recreate();
```
Comment (frenzibyte) [inline osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs:123]: iirc BDLs with expression bodies were not wanted, statement bodies look better for these cases.
```suggestion
        private void load() 
        {
             recreate();
        }
```

Comment (bdach) [reply osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs:123]: this comment is irrelevant. a method is a method is a method. and there are even [existing examples](https://github.com/ppy/osu/blob/d8d2c731ee42db1ecc7e1a6c8cb3d5f1dad8b943/osu.Game/Screens/Play/HUD/DefaultComboCounter.cs#L26-L27) of expression-bodied BDLs.

Comment (frenzibyte) [reply osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs:123]: this comment is not irrelevant. I was mistakingly searching parameterless BDL so couldn't catch one, but I'm 99.99% sure I was suggested to always use statement bodies for BDL, could be my memory not serving me.

**Review by bdach — APPROVED** [review]
seems fine

**Pull request merged**